src/core/evaluator.js

```diff
@@ -2212,6 +2212,12 @@ class PartialEvaluator {
               })
             );
             return;
+          case OPS.setLineWidth:
+            // The thickness should be a non-negative number, as per spec.
+            // When the value is negative, Acrobat and Poppler take the absolute
+            // value while PDFium takes the max of 0 and the value.
+            args[0] = Math.abs(args[0]);
```

Comment:
We should probably also check that it's a number, and ignore it otherwise (with a warning?) to prevent issues?

Reply:
The lineWidth can also be set in a /ExtGState dictionary, hence that case needs to be handled as well:
https://github.com/mozilla/pdf.js/blob/d008452e80ca00eec32983b7ab139885e6bc9f56/src/core/evaluator.js#L1124